swarm/docker/Dockerfile

```diff
@@ -10,14 +10,23 @@ RUN mkdir -p $GOPATH/src/github.com/ethereum && \
     git checkout ${VERSION} && \
     go install -ldflags "-X main.gitCommit=${VERSION}" ./cmd/swarm && \
     go install -ldflags "-X main.gitCommit=${VERSION}" ./cmd/swarm/swarm-smoke && \
-    go install -ldflags "-X main.gitCommit=${VERSION}" ./cmd/geth && \
-    cp $GOPATH/bin/swarm /swarm && cp $GOPATH/bin/geth /geth && cp $GOPATH/bin/swarm-smoke /swarm-smoke
+    go install -ldflags "-X main.gitCommit=${VERSION}" ./cmd/swarm/global-store && \
+    go install -ldflags "-X main.gitCommit=${VERSION}" ./cmd/geth
```

Comment:
I am not sure I understand what's being changed here.

The previous code was doing a `cp` of `swarm` and `geth` to `/` - why is that not needed anymore? I remember that the image was relying somehow on this...